MyFirstSlideshow/app/src/main/java/com/charles/myfirstslideshow/MainActivity.java

```diff
@@ -10,12 +26,141 @@
  *  This code is for demonstration purposes only. It should not be assumed to represent best
  *  practices or patterns employed at Yoti.
  */
+public class MainActivity extends AppCompatActivity implements Serializable {
+
+    // Images under Creative Commons
+    // Images attributed to (in order) https://www.flickr.com/photos/_torne/
+    // https://www.flickr.com/photos/chrisyarzab/
+    // https://www.reddit.com/user/lalien42/
+    // http://www.kapstadt.de/
+
+    public static List images = Arrays.asList(new String[]{"https://c1.staticflickr.com/6/5615/15570202337_0e64f5046e_k.jpg",
```

Comment:
You can specify arrays in resources file. It will be better. Check out the documentation: 
https://developer.android.com/guide/topics/resources/available-resources